### PR TITLE
Updating to use date-fns for job creation, update, and start.

### PR DESCRIPTION
**Pull request by jvdub**
Because I can't add tasks, I can't test this locally; however, in the testing I was able to do in other ways, I _think_ this should fix issue #8... Only way to know is to have the maintainer run it on the data they have. I hope this helps!

### CODE REVIEW

File: src/components/JobColumn.vue [hunk 1]
```diff
@@ -28,9 +28,9 @@
         v-for="(job, index) in jobs" :key="job.id"
         :jobId="job.id"
         :jobType="job.type"
-        :createdAt="job.created_at"
-        :updatedAt="job.updated_at"
-        :startedAt="job.started_at"
+        :createdAt="format(job.created_at, 'DD/MM/YYYY HH:mm:ss')"
```
Comment (delete) [inline src/components/JobColumn.vue:31]: We can't access the **format** function here, directly in the template. We must "bind" an imported function to a variable in the _Vue reactive scope_ or create an another function as a wrapper.

**Comment by delete** [comment]
Hey @jvdub, thanks for the contribution! :smile: 

As I mention in the comment, we could create a function in the Vue component to work as a wrapper, something like this...


```ts
public formatDate(dateString: string): string {
    const date = new Date(Number(dateString));
    return format(date, 'DD/MM/YYYY HH:mm:ss ');
  }
```

Something I noticed testing your PR, the all job dates is in _unix timestamp_ and if I'm not wrong, date-fns doesn't have support yet. :disappointed: 

I think we should convert the dateString first, using the Date creator.

**Comment by delete** [comment]
Can I merge and add the changes or do you want to try?

**Comment by jvdub** [comment]
I probably won't have time for another few days, so go for it!

**Pull request merged**